netmap/netmap_contract.go

```diff
@@ -396,7 +430,7 @@ func LastEpochBlock() int {
 // of current epoch.
 func Netmap() []storageNode {
 	ctx := storage.GetReadOnlyContext()
-	return getSnapshot(ctx, snapshot0Key)
+	return getSnapshot(ctx, snapshotKeyPrefix+"0")
```

Comment:
Shouldn't it be `snapshotKeyPreifx+storage.Get(snapshotCurrentIDKey)`?

Reply:
Fixed.